### DISCUSSION TITLE
Top languages compact layout hides the progress bar, then when switching back and forth tabs it magically appears

### DISCUSSION THREAD

**Discussion by ppazos** [anuraghazra/github-readme-stats#3664] [Q&A] [answered]
### Describe the bug

Testing https://github-readme-stats.vercel.app/api/top-langs/?username=ppazos&theme=dark&langs_count=6&layout=compact

It doesn't show the progress bar.

Then I switch to another tab, then I go back to that one and the progress bar is there :)

### Expected behavior

The progress bar should be there since it's not hidden.

### Screenshots / Live demo link

https://github-readme-stats.vercel.app/api/top-langs/?username=ppazos&theme=dark&langs_count=6&layout=compact

**Comment by qwerty541** [comment, marked as answer]
Hey, @ppazos! Thanks for your report, but this is not a bug in our code, looks like you experiencing some kind of animation freezing in your browser. We recently discussed this in https://github.com/anuraghazra/github-readme-stats/issues/3487 with another collaborators and we came to conclusion that we cannot handle this from our side. If you experience this problem too often than you can disable animations with `&disable_animations=true` query option. If you have any ideas about how we can deal with this issue from our side feel free to join the discussion and share.

**Reply by ppazos** [reply]
Thanks @qwerty541 I should have reported that I'm on Firefox Linux, so not sure if the issue with the animations is browser or OS specific.

I can confirm this works as expected https://github-readme-stats.vercel.app/api/top-langs/?username=ppazos&theme=dark&langs_count=6&layout=compact&disable_animations=true

Thanks!
